autoload/lsp/utils/text_edit.vim

```diff
@@ -14,6 +14,24 @@ function! lsp#utils#text_edit#apply_text_edits(uri, text_edits) abort
     endif
 endfunction
 
+function! lsp#utils#text_edit#build_loclist_items(uri, text_edits) abort
```

Comment:
Should name the function similar to https://github.com/prabirshrestha/vim-lsp/blob/51eb1f6676d914b059d2a7ec7cadc66f84ef156e/autoload/lsp/utils/location.vim#L115

this the new coding guidelines. `lsp#utils#location#_lsp_to_vim_list`

Reply:
can we not reuse https://github.com/prabirshrestha/vim-lsp/blob/36cc10476489c61c0c79d1e68784beddf5efc879/autoload/lsp/utils/location.vim#L115?

Reply:
I guess, but the interface is a bit different:

```ts
interface TextEdit {
	/**
	 * The range of the text document to be manipulated. To insert
	 * text into a document create a range where start === end.
	 */
	range: Range;

	/**
	 * The string to be inserted. For delete operations use an
	 * empty string.
	 */
	newText: string;
}
```

```ts
interface Location {
	uri: DocumentUri;
	range: Range;
}
```

Reply:
then it make sense to not reuse. we should try to reuse the naming convention.